test/parallel/test-debug-brk.js

```diff
@@ -65,6 +65,6 @@ function test(extraArgs, stdoutPattern) {
 
 test(['-e', '0']);
 test(['-e', '0', 'foo']);
-test(['-p', 'process.argv[1]', 'foo'], /^\s*foo\s*$/);
+test(['-p', 'process.argv[1]', 'foo']);
```

Comment:
You're leaving behind dead code with this change.  Remove `stdoutPattern` and `outputMatched` from test().  There may be more.